libs/designsystem/src/lib/components/checkbox/checkbox.component.scss

```diff
@@ -107,6 +107,11 @@ $spacing-to-label: map-get($checkbox-radio-spacing, 'to-label');
       flex-shrink: 0;
       flex-grow: 0;
     }
+
+    span {
+      align-self: center;
+      line-height: line-height('n');
```

Comment:
Should the line-height be set on the entire comp instead? 🤔

Reply:
resolved by: a8e1e31